packages/eslint-plugin-nx/src/rules/enforce-module-boundaries.ts

```diff
@@ -228,6 +238,95 @@ export default createESLintRule<Options, MessageIds>({
           data: {
             npmScope,
           },
+          fix(fixer) {
+            const targetImportProject = getTargetProjectBasedOnRelativeImport(
```

Comment:
We are running the `isRelative` twice - as part of `isRelativeImportIntoAnotherProject` and again at `getTargetProjectBasedOnRelativeImport`. 

I think we should split the condition on line 225 to check for relative and absolute import separately. That way we don't have to check for targetProjectImport twice.